tests/test_map_ap_data.py

```diff
@@ -65,26 +64,28 @@ def make_input_source_catalog(n_objects, add_flags=False):
         schema.addField("base_PixelFlags_flag", type="Flag")
         schema.addField("base_PixelFlags_flag_offimage", type="Flag")
 
-    objects = afwTable.SourceCatalog(schema)
-    objects.preallocate(n_objects)
-    objects.definePsfFlux("base_PsfFlux")
-    objects.defineCentroid("base_NaiveCentroid")
+    exposure, catalog = dataset.realize(10.0, schema, randomSeed=1234)
+    catalog.definePsfFlux("base_PsfFlux")
+    catalog.defineCentroid("base_NaiveCentroid")
 
-    for obj_idx in range(n_objects):
-        obj = objects.addNew()
+    for src in catalog:
         for subSchema in schema:
-            if isinstance(obj.get(subSchema.getKey()), geom.Angle):
-                obj.set(subSchema.getKey(), 1. * geom.degrees)
-            elif subSchema.getField().getName() == "ip_diffim_DipoleFit_neg_instFlux":
-                obj.set(subSchema.getKey(), -1)
-            else:
-                obj.set(subSchema.getKey(), 1)
-    return objects
+            if subSchema.getField().getName() == "ip_diffim_DipoleFit_neg_instFlux":
```

Comment:
I realise the logic here is mostly not new, just rearranged, but... I kinda feel like a comment about what's going on would be helpful. Why are these particular fields being set to these particular values?